### DISCUSSION TITLE
CloudFlare costs

### DISCUSSION THREAD

**Discussion by homerjam** [soketi/pws#862] [Cloudflare Workers: Support]
Hi, we've been using Soketi on CloudFlare for a while and it's been great. However our most recent bill seems to have spiked due to durable object usage.

Is this expected? Is there anything to be aware of when using Soketi to keep costs down?

Thanks!

**Comment by tbell511** [comment]
How much have you been paying? And how many events / concurrent users do you typically have?

**Reply by fabioselau077** [reply]
ward

**Comment by rennokki** [comment]
The pricing varies. Serverless is more expensive at high load because the business use case is to cut down costs of maintaining servers if the usage is infrequent.
